sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorClientBuilder.java

```diff
@@ -421,9 +481,17 @@ public EventProcessorClient buildEventProcessorClient() {
         }
 
         final TracerProvider tracerProvider = new TracerProvider(ServiceLoader.load(Tracer.class));
+        if (loadBalancingUpdateInterval == null) {
+            loadBalancingUpdateInterval = DEFAULT_LOAD_BALANCING_UPDATE_INTERVAL;
+        }
+        if (partitionOwnershipExpirationInterval == null) {
```

Comment:
In another `EventProcessorClientBuilder` I see "@throws NullPointerException if {@code partitionOwnershipExpirationInterval} is {@code null}". Could this be Null?

Reply:
Yes, the `partitionOwnershipExpirationInterval` is null by default. If the user attempts to set this using `public EventProcessorClientBuilder partitionOwnershipExpirationInterval(Duration partitionOwnershipExpirationInterval)` method, then we check that the user-provided value is not null/negative/zero.